combinations/hash.tsv

```diff
@@ -335,3 +335,4 @@ r-base,r-aod,r-ggplot2,r-plyr
 bbmap=38.98,seqkit=2.3.1,pigz=2.6
 gatk4=4.3.0.0,gatktool=0.0.1
 fgbio=2.0.2,bwa=0.7.17,samtools=1.16.1
+wham==1.8.0,tabix==1.11
```

Comment:
why `==`? all other combinations seem to use a single `=`

Reply:
Oh good spot, how I missed that :face_with_spiral_eyes: